# cmpc_controller parameters

max_thrust: 20.0

min_thrust: 2.0

max_bodyrate: 3.0

# weight matrix of state costs
Q: [200.0, 200.0, 500.0, 50, 50, 50, 50, 10, 10, 10 ]
# Q: [0.5, 0.5, 0.5, 0.1, 0.1, 0.1, 0.1, 0.05, 0.05, 0.05 ]
#位置，姿态（四元数），速度

# weight matrix of control effort
R: [ 1, 1, 1, 1 ]
